src/ServerConfig.cc

```diff
@@ -819,28 +819,28 @@ ignition::gazebo::parsePluginsFromString(const std::string &_str)
 
 /////////////////////////////////////////////////
 std::list<ServerConfig::PluginInfo>
-ignition::gazebo::loadPluginInfo(bool _isPlayback)
+gazebo::loadPluginInfo(bool _isPlayback)
 {
   std::list<ServerConfig::PluginInfo> ret;
 
   // 1. Check contents of environment variable
   std::string envConfig;
-  bool configSet = ignition::common::env(gazebo::kServerConfigPathEnv,
```

Comment:
If the `gazebo` namespace is still valid, wouldn't it be better to leave `gazebo::kServerConfigPathEnv` intact?

Reply:
We're trying to future-proof to reduce the diff for when we change to `sim`